src/components/Pagination/styled/pageButton.js

```diff
@@ -16,7 +17,7 @@ const StyledPageButton = styled.li`
 
     :nth-last-child(2) > button {
         margin-right: ${MARGIN_X_SMALL};
-        border-radius: 0 100px 100px 0;
+        border-radius: ${props => getBorderRadius(props.pages)};
```

Comment:
There should be an approach that use only CSS, if not possible we can simply use:
```
function getBorderRadius() {
    if (pages === 1) {
        return '100px';
    }
    return '0 100px 100px 0';
}
 ```